Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/WPF/InkCanvas/InkCanvas.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Windows.Interop;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.WPF
+{
+    public class InkCanvas : WindowsXamlHost
```

Comment:
>WindowsXamlHost [](start = 29, length = 15)

I think Ryland was adding a WindowsXamlHostBase for the wrapped controls to derive from.